winui/Shimmer/Custom-Layout.md

```diff
@@ -0,0 +1,325 @@
+};
+Grid.SetRow(rectangle4, 2);
+Grid.SetColumn(rectangle4, 1);
+
+Rectangle rectangle5 = new Rectangle()
+{      
+    RadiusX = 3,
+    RadiusY = 3,
+    HorizontalAlignment = HorizontalAlignment.Left,
+    Height = 10,
+    Width = 190,
+};
+Grid.SetRow(rectangle5, 3);
+Grid.SetColumn(rectangle5, 1);
+
+Rectangle rectangle6 = new Rectangle()
+{     
+    RadiusX = 3,
+    RadiusY = 3,
+    HorizontalAlignment = HorizontalAlignment.Left,
+    Height = 10,
+    Width = 170,
+};
+Grid.SetRow(rectangle6, 4);
+Grid.SetColumn(rectangle6, 1);
+
+Rectangle rectangle7 = new Rectangle()
+{      
+    RadiusX = 3,
+    RadiusY = 3,
+    HorizontalAlignment = HorizontalAlignment.Left,
+    Height = 10,
+    Width = 190,
+};
+Grid.SetRow(rectangle7, 5);
+Grid.SetColumn(rectangle7, 1);
+
+Rectangle rectangle8 = new Rectangle()
+{      
+    RadiusX = 3,
+    RadiusY = 3,
+    HorizontalAlignment = HorizontalAlignment.Left,
+    Height = 10,
+    Width = 150,
+};
+Grid.SetRow(rectangle8, 6);
+Grid.SetColumn(rectangle8, 1);
+
+Rectangle rectangle9 = new Rectangle()
+{     
+    RadiusX = 3,
+    RadiusY = 3,
+    HorizontalAlignment = HorizontalAlignment.Left,
+    Height = 10,
+    Width = 160,
+};
+Grid.SetRow(rectangle9, 7);
+Grid.SetColumn(rectangle9, 1);
+
+Rectangle rectangle10 = new Rectangle()
+{    
+    RadiusX = 3,
+    RadiusY = 3,
+    HorizontalAlignment = HorizontalAlignment.Left,
+    Height = 10,
+    Width = 175,
+};
+Grid.SetRow(rectangle10, 8);
+Grid.SetColumn(rectangle10, 1);
+
+Rectangle rectangle11 = new Rectangle()
+{     
+    RadiusX = 3,
+    RadiusY = 3,
+    HorizontalAlignment = HorizontalAlignment.Left,
+    Height = 10,
+    Width = 160,
+};
+Grid.SetRow(rectangle11, 10);
+
+Rectangle rectangle12 = new Rectangle()
+{     
+    RadiusX = 3,
+    RadiusY = 3,
+    HorizontalAlignment = HorizontalAlignment.Left,
+    Height = 30,
+    Width = 175,
+};
+Grid.SetRow(rectangle12, 10);
+Grid.SetColumn(rectangle12, 1);
+Grid.SetRowSpan(rectangle12, 12);
+
+Rectangle rectangle13 = new Rectangle()
+{     
+    RadiusX = 3,
+    RadiusY = 3,
+    HorizontalAlignment = HorizontalAlignment.Left,
+    Height = 10,
+    Width = 175,
+};
+Grid.SetRow(rectangle13,11);
+
+customLayout.Children.Add(rectangle1);
+customLayout.Children.Add(rectangle2);
+customLayout.Children.Add(rectangle3);
+customLayout.Children.Add(rectangle4);
+customLayout.Children.Add(rectangle5);
+customLayout.Children.Add(rectangle6);
+customLayout.Children.Add(rectangle7);
+customLayout.Children.Add(rectangle8);
+customLayout.Children.Add(rectangle9);
+customLayout.Children.Add(rectangle10);
+customLayout.Children.Add(rectangle11);
+customLayout.Children.Add(rectangle12);
+customLayout.Children.Add(rectangle13);
+
+sfShimmer.CustomLayout = customLayout;
+
+Grid mainGrid = new Grid();
+mainGrid.Children.Add(sfShimmer);
+this.Content = mainGrid;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Shimmer Custom Layout in WinUI](Shimmer_images/winui_shimmer_customlayout.gif)
```

Comment:
@GilbertmanojSF3971 Custom layout in WinUI Shimmer.

Reply:
@Manivannan-E Sir, Changed the image Title name.